.travis.yml

```diff
@@ -34,5 +34,15 @@ script:
 after_success:
     - if [[ $TRAVIS_PYTHON_VERSION != pypy* ]]; then pip install coveralls --use-mirrors ; coveralls ; fi
 
+# publish new version to pypi
+deploy:
+  provider: pypi
```

Comment:
If desired, you could test this first with [Test PyPI](https://packaging.python.org/guides/using-testpypi/):

https://docs.travis-ci.com/user/deployment/pypi/#releasing-to-a-self-hosted-pypi


E.g. I think that would be:

```
server: https://test.pypi.org/simple/
```